smaug/python/graph.py

```diff
@@ -77,6 +77,9 @@ def add_node(
         node.src_tensors_indices.append(tensor.source[1])
         if node.name not in tensor.source[0].children:
           tensor.source[0].children.append(node.name)
+      else:
+        node.parents.append("N/A")
```

Comment:
Doesn't N/A just mean a node has no parents? If so, why do we need to add fake parents?

Reply:
As explained in a previous comment, we need an indicator to let the C++ side know that this input tensor doesn't have a source operator.